pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -32,11 +33,24 @@ const Name = "NodeLabel"
 
 // Args holds the args that are used to configure the plugin.
 type Args struct {
-	// The list of labels that identify node "groups"
-	// All of the labels should be either present (or absent) for the node to be considered a fit for hosting the pod
-	Labels []string `json:"labels,omitempty"`
-	// The boolean flag that indicates whether the labels should be present or absent from the node
-	Presence bool `json:"presence,omitempty"`
+	// PresentLabels should be present for the node to be considered a fit for hosting the pod
+	PresentLabels []string `json:"presentLabels,omitempty"`
+	// AbsentLabels should be absent for the node to be considered a fit for hosting the pod
+	AbsentLabels []string `json:"absentLabels,omitempty"`
+}
+
+// validateArgs validates that PresentLabels and AbsentLabels do not conflict.
+func validateArgs(args *Args) error {
+	presentLabels := make(map[string]struct{}, len(args.PresentLabels))
+	for _, l := range args.PresentLabels {
+		presentLabels[l] = struct{}{}
+	}
+	for _, l := range args.AbsentLabels {
+		if _, ok := presentLabels[l]; ok {
+			return fmt.Errorf("detecting at least one label (e.g., %q) that exist in both the present and absent label list: %+v", l, args)
+		}
+	}
```

Comment:
can you convert this into a function that accepts two lists, presentLabels and absentLabels because we can reuse it to verify the label lists of the score-based plugin.